src/instructlab/config/init.py

```diff
@@ -176,8 +176,7 @@ def prompt_user_to_choose_vendors(
 
     system_profile_selection = click.prompt(
         "Enter the number of your choice",
-        type=int,
-        default=0,
```

Comment:
What happens if you set a default with IntRange also set?  Does is not still loop if you pick an invalid value?  It seems like was the original desired behavior.

Reply:
This may be an overreach on my part but removing the default requires the user to be explicit in the HW family choice.

Reply:
I don't have a strong opinion.  Wanted to note it is a behavior change.

Reply:
I restored the `default=0` after considering upstream/downstream documentation would need to be updated for that minor change.